File: tests/fixtures/code_samples/ts/sample_054.ts
/** Sample 54: small utility. */

export function operation54(xs: number[]): number {
  let total = 54;
  for (const x of xs) total += x;
  return total;
}

export function operationPure54(value: number): number {
  return (value * 54) % 7919;
}

File: tests/fixtures/code_samples/py/sample_054.py
"""Sample 54: small utility function."""

from __future__ import annotations
from typing import Iterable


def operation_54(xs: Iterable[int]) -> int:
    """Return the sum of `xs` plus the index marker 54."""
    total = 54
    for x in xs:
        total += int(x)
    return total


def operation_54_pure(value: int) -> int:
    return (value * 54) % 7919

